tools/cluster-deploy/cluster-deploy.go

```diff
@@ -22,10 +23,14 @@ func main() {
 		log.Fatal("--ocs-catalog-image is required")
 	} else if *ocsSubscriptionChannel == "" {
 		log.Fatal("--ocs-subscription-channel is required")
+	} else if *installNamespace == "" {
```

Comment:
I think this is not required.

Reply:
It's required, as someone may pass the flag but it will not be set or empty. In that case & an empty namespace will generate runtime segfault errors.